googlesource/googlesource.go

```diff
@@ -55,53 +47,55 @@ func (client *client) Client() (*bigquery.Client, error) {
 }
 
 // checkInitialPos helps in creating the query to fetch data from endpoint
-func (s *Source) checkInitialPos(positions map[string]string, incrementColName map[string]string, tableID string, primaryColName map[string]string) (firstSync, userDefinedOffset bool, userDefinedKey bool) {
+func (s *Source) checkInitialPos(positions, incrementColName, primaryColName string) (firstSync, userDefinedOffset, userDefinedKey bool) {
```

Comment:
All of the arguments can be taken from `s`, so we actually don't need any parameters for this method.